templates/module/template.yml

```diff
@@ -0,0 +1,2 @@
+description: "Execution module"
+version: "Carbon"
```

Comment:
Instead of hardcoding the version name, consider getting it at runtime:
```python
Python 2.7.11 (default, Apr 29 2016, 18:08:53) 
[GCC 5.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import salt.version
>>> salt.version.__saltstack_version__.name
'Boron'
>>> 
```
Or even better, since we're after the next release name:
```diff
index d02a49e..3d351d0 100644
--- a/salt/version.py
+++ b/salt/version.py
@@ -275,6 +275,16 @@ class SaltStackVersion(object):
             ]
         )
 
+    @classmethod
+    def next_release(cls):
+        return cls.from_name(
+            cls.VNAMES[
+                min([version_info for version_info in
+                     cls.VNAMES if
+                     version_info > cls.from_last_named_version().info])
+            ]
+        )
+
     @property
     def sse(self):
         # Higher than 0.17, lower than first date based
```
```python
Python 2.7.11 (default, Apr 29 2016, 18:08:53) 
[GCC 5.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import salt.version
>>> salt.version.SaltStackVersion.next_release()
<SaltStackVersion name='Carbon' major=9223372036854775704 minor=0 bugfix=0>
>>> salt.version.SaltStackVersion.next_release().name
'Carbon'
>>> 
```